R/tidyme.R

```diff
@@ -87,9 +87,21 @@ tidyme.survfit <- function(x, ...) {
       retme[["n.strata"]] <- rep(x[["n"]], x[["strata"]])
     }
   }
-  
+
   attr(retme, "survfit_object") <- survfit_object
+  strata <- .extract_strata_varlist(survfit_object)
```

Comment:
where is this function defined? I couldnt find it

Reply:
In RStudio, you can find any function object by pressing `Ctrl+.`

Reply:
I know. I meant that there was no function defined in the files :)

Reply:
moved to utils file